src/abis/IVexchangeV2Pair.ts

```diff
@@ -1,4 +1,4 @@
-{
+export default {
```

Comment:
Generally prefer named exports, objections to:
```suggestion
export PAIR_ABI = {
```

Reply:
fixed